### DISCUSSION TITLE
Estimate number of keys = 0

### DISCUSSION THREAD

**Discussion by mark-e-hoffman** [apache/kvrocks#1972] [Q&A]
the info cmd returns zero for the estimate number of keys. I also tried directly accessing this property with a rocksdb library and it also returns 0. I've also tried to iterate the db with some rocksdb code and that also returns no rows? How can I get a decent estimate on the number of keys in my database?
Thanks

**Comment by git-hulk** [comment]
Hi @mark-e-hoffman 

Could you please paste your info command output? the estimated number of keys in the INFO command section came from the rocksdb, so it means no keys in rocksdb if this value is zero.
